pages/api/enroll/child.ts

```diff
@@ -12,6 +12,10 @@ import { ParentRegistrationInput } from "models/Enroll";
 import { roles } from "@prisma/client";
 import { getUserFromEmail } from "@database/user";
 import { getClass } from "@database/class";
+import { getWaitlistRecordsByClassId } from "@database/waitlist";
+import send from "services/nodemailer/mail";
+import { openSpotWaitlistTemplate } from "@utils/mail/templateWaitlist";
+import { IoRecordingOutline } from "react-icons/io5";
```

Comment:
@aamyli Please try to remove github action warning when ever possible. It doesn't look like this icon is being used.